### ISSUE TITLE
keyboard navigation with screen reader NVDA

### ISSUE THREAD

**Issue by Wojtaszek104** [N00ts/vue3-treeview#43]
screen readers that meet the wcag conditions cannot fully use treeview because after expanding the tree, lower elements still have tabindex = -1, which makes it impossible to control and further expand the elements

**Comment by N00ts** [comment]
I'm affraid I will not have time soon to fix that bug, if you want you can create a PR  and I will have a look at it :)
